cmd/lnmuxd/run.go

```diff
@@ -126,6 +130,18 @@ func runAction(c *cli.Context) error {
 		return err
 	}
 
+	promAddress := cfg.PrometheusAddress
+	if promAddress == "" {
+		promAddress = DefaultPrometheusAddress
+	}
+
+	// Instantiate a new HTTP server and mux.
+	metrics := &http.Server{
+		Addr:              promAddress,
+		Handler:           promhttp.Handler(),
+		ReadHeaderTimeout: 10 * time.Second,
```

Comment:
Comment as to why this is necessary?

Reply:
I added a comment. It's not quite as relevant for a server where inbound is ostensibly only from trusted clients but IMO defense in depth is better than turning off the linter warning.

Reply:
Ah, there was a linter warning. Interesting.